Assignment_0/210456_Ishan.md

```diff
@@ -0,0 +1,74 @@
+# HackIT
+
+## UNIX file system
+
+-UNIX file system provides with a way to reach any file on one's PC using Command line interface, it is divided into directories and uses a tree interface to simplify things further
+
+-some useful commands to navigate and interact are:
+
+    cd-command to enter root directory
+        cd.. :to go back to previous directory
+        cd/ : to go to home directory
```

Comment:
`cd /` goes to root, `cd ~` or simply `cd` will take you to home. Also please add spaces in between.